.github/workflows/scripts/build.sh

```
@@ -0,0 +1,29 @@
+        echo "Invalid version format: $version"
+    fi
+}
+
+function get_branch() {
+    local version_file="version"
+    if [[ ! -f $version_file ]]; then
+        echo "Error: Version file '$version_file' not found."
+        exit 1
+    fi
+
+    local version=$(cat "$version_file")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)